l5kit/l5kit/tests/simulation/e2e_test.py

```diff
@@ -0,0 +1,122 @@
+from l5kit.cle.validators import RangeValidator, ValidationCountingAggregator
+from l5kit.dataset import EgoDataset
+from l5kit.simulation.dataset import SimulationConfig
+from l5kit.simulation.unroll import ClosedLoopSimulator
+from l5kit.tests.simulation.unroll_test import MockModel
+
+
+@pytest.fixture(scope="function")
+def simulation_evaluator() -> ClosedLoopEvaluator:
+    return ClosedLoopEvaluator(EvaluationPlan(metrics=[DisplacementErrorL2Metric(),
+                                                       DistanceToRefTrajectoryMetric(),
+                                                       CollisionFrontMetric(),
+                                                       CollisionRearMetric(),
+                                                       CollisionSideMetric()],
+                                              validators=[RangeValidator("displacement_error_l2_validator",
+                                                                         DisplacementErrorL2Metric, max_value=30),
+                                                          RangeValidator("distance_ref_trajectory_validator",
+                                                                         DistanceToRefTrajectoryMetric, max_value=4),
+                                                          RangeValidator("collision_front_validator",
+                                                                         CollisionFrontMetric, max_value=0),
+                                                          RangeValidator("collision_rear_validator",
+                                                                         CollisionRearMetric, max_value=0),
+                                                          RangeValidator("collision_side_validator",
+                                                                         CollisionSideMetric, max_value=0),
+                                                          ],
+                                              composite_metrics=[],
+                                              intervention_validators=["displacement_error_l2_validator",
+                                                                       "distance_ref_trajectory_validator",
+                                                                       "collision_front_validator",
+                                                                       "collision_rear_validator",
+                                                                       "collision_side_validator"]))
+
+
+def test_e2e_no_models(ego_cat_dataset: EgoDataset, simulation_evaluator: ClosedLoopEvaluator) -> None:
+    sim_cfg = SimulationConfig(use_ego_gt=True, use_agents_gt=True, disable_new_agents=False,
+                               distance_th_far=1, distance_th_close=0)
+
+    sim_loop = ClosedLoopSimulator(sim_cfg, ego_cat_dataset, torch.device("cpu"))
+    sim_out = sim_loop.unroll(list(range(len(ego_cat_dataset.dataset.scenes))))
```

Comment:
Can you put this into a fixture (module scope) that returns the simulation outputs ? So that we can re-use on all other tests.